composer.json

```diff
@@ -27,7 +27,7 @@
         "spatie/once": "^3.1"
     },
     "require-dev": {
-        "orchestra/testbench": "^8.0",
+        "orchestra/testbench": "^8.5.10",
```

Comment:
This version is the lowest to support Laravel Framework `10.15`: https://github.com/orchestral/testbench/releases/tag/v8.5.10